src/components/Card.js

```diff
@@ -9,12 +9,15 @@ import BarChart from './BarChart';
 
 const cardStyle = makeStyles(theme => ({
   root: {
+    padding: '0 0 16px 0'
+  },
+  cardProperty: {
     padding: theme.spacing(1, 1)
   }
 }));
 
 const Card = ({ content: { title, body, type } }) => {
-  const { root: paperClasses } = cardStyle();
+  const classes = cardStyle();
```

Comment:
This is fine. Now that I'm seeing that you're using destructuring a lot more, I'll lay off, unless it's a situation that _really_ merits using it. 🌮